README.md

```diff
@@ -1,29 +1,56 @@
+
+## Testing and building locally
+Tests for the CI-server are written using the `JUnit` library. If you want to run the tests locally, there are two ways.
+* **Using the terminal:** first navigate to the the project folder, then use the command `./gradlew test`. To build the project locally use the command `./gradlew build`. For windows machines you can use `gradlew test/build`.
+* **Using IntelliJ:** Open the project in IntelliJ. Right-click on the src-folder and choose `Run 'Tests in DD2480-Big-Brain-CI.test'`. To build the project right-click on the src-folder and choose  `Build module DD2480-Big-Brain-CI`.
+If the build was successful there will be a new directory in the project named 'Build' that contains the generated classes. If it does not succeed the directory will not contain any generated classes.
+
+## Managing the CI-server
+### Deploy the server
+Using [ngrok](https://ngrok.com/) we can make the server visible on the internet. 
+If you don't have it installed write the following in the terminal:
+
+```
+#For linux users: 
+curl -LO --tlsv1 https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
+unzip ngrok-stable-linux-amd64.zip 
+#For Mac users:
+curl -LO --tlsv1 https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-darwin-386.zip
+unzip ngrok-stable-darwin-386.zip
+```
 
-## Using the CI-server
-1. Go to the assessment branch
+All you have to do is open a new terminal window and use the command `./ngrok http 8083`. The ngrok service provides a forwarding URL that works for 8 hours.
+
+After this you have to set up a Webhook. This can be done on the Github repository online.
+1. Go to `Settings >> Webhooks`, click on `Add webhook`.
+1. Paste the forwarding URL that `ngrok` provides (for example `https://7al9b010.ngrok.io`) in the payload field and proceed to click `Add webhook`. Nothing else is required than the URL.
+
+Now the CI-server should be ready to be used. 
+
+### Using the CI-server
+To test the CI-server there is a specific branch prepared. Follow the following steps in the terminal to proceed. These steps assumes you have already cloned the repository.
+1. Go to the assessment branch using the command `git checkout -b assessment origin/assessment`
 1. Make a change somewhere
-1. Push changes to `GitHub`
-1. The CI-server will do its part and notify about the results
+1. Commit and push the changes to `GitHub`
+1. The CI-server will do its part and notify about the results of the build and tests using Slack
 
-## What the server is doing
-1. Clone the project using `JGit`, this happens when webhook sees a change (such as a push) in the repository
-1. Compile and run the test using `Gradle` and the Gradle tooling API `GradleConnector` (can be done using the command `gradlew build` in the terminal)
-1. Notify the user using the `XML-file` generated by `Gradle`
+## When the server is activated by the Webhook
+1. Clones the project using `JGit`and saves it to a directory named 'assessment'.
```

Comment:
It doesn't use `JGit` anymore. I'm only sending command line from Java
It saves to a directory named `assessmentDir`